src/components/thead/thead-styles.jsx

```diff
@@ -6,41 +6,51 @@ export default theme => {
 
   return {
     thead: {
+      width: '100%',
+      ...styleUtils.sizes(),
       ...mixins.basicBoxSizing,
-      display: 'block',
       fontWeight: typography.fontWeightSemiBold,
       borderColor: color.gray,
       borderBottom: `2px solid ${palette.shades.dark.text.muted}`,
-      ...styleUtils.sizes(),
-      width: '100%',
-      fontSize: 12,
-
-      '&$hiddenOnMobile': {
-        display: 'none',
 
-        [mixins.media('md')]: {
-          display: 'block',
-        },
+      '&.primary': {
```

Comment:
If both primary and secondary has `borderBottom: 0` maybe it can be moved out of the class scopes?

Reply:
Ill combine them, there is 3 modes..  undefined, primary and secondary